src/components/CardView.js

```diff
@@ -0,0 +1,56 @@
+import React, {Component} from "react";
+import {Header} from "semantic-ui-react";
+import {Helmet} from "react-helmet";
+import {Map, TileLayer, Marker, Popup, Polyline} from "react-leaflet";
+import {connect} from "react-redux";
+
+const zoomLevel = 13;
+
+class Card extends Component {
+    render() {
+        const center = [51.505, -0.09]
+        const {gpsData} = this.props;
+        const start = gpsData[0];
+        const end = gpsData[gpsData.length - 1];
+        const array = [];
+        gpsData.map(element => {
+            array.push([element.latitude, element.longitude]);
+        });
+        console.log(array);
+        console.log(start);
+        const data = [[51.505, -0.09], [51.51, -0.1], [51.51, -0.12]];
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this useless assignment to local variable "data" [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=javascript%3AS1854)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the declaration of the unused 'data' variable. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=javascript%3AUnusedVariable)